pyproject.toml

```diff
@@ -52,5 +52,25 @@ build-backend = "setuptools.build_meta"
 version_scheme = "post-release"
 local_scheme = "node-and-date"
 
+[tool.isort]
```

Comment:
Excellent!

Reply:
I've really liked having `isort` in there ... it's the kind of thing where I just put an import somewhere up top and I don't have to think about it at all!  `isort` just puts it in the right place.  Although it did take a little fiddling to define what the correct place is, since there's apparently some variety across python projects.